README.md

```diff
@@ -128,5 +128,16 @@ The below checklist should be complete to ensure a story is ready.
 - [ ] The user story is in line with the Product Vision
 - [ ] All blockers and dependencies for the needed implementation are resolved
 
+# Sprint 1 Burndown Chart
+https://docs.google.com/spreadsheets/d/1wAimr26wow8j6nflQbo_E_7oXwHJKrT59--CRPgGTjE/edit#gid=1432140175
+
+# Sprint 1 Evidence of Daily Scrum
+https://github.com/jorgecotillo/green_light_health/blob/master/img/july-2-2020/scrum/sprint1-scrum1.png
```

Comment:
Can you split this README work into a separate pull request? The evidence of daily scrums, sprint planning, etc is within the 'img' directory at the root of the repository. We should link to that instead. The reason it should be two PRs is because else you are asking for two different types of work to be merged at once, which tends to be problematic with reviewers/comments.

Reply:
I'll decline this pr and rework tomorrow